### ISSUE TITLE
Profile cannot be exported on Windows if "make Tox portable" is on

### ISSUE THREAD

**Issue by rubo77** [qTox/qTox#849]
I selected "Make Tox Portable"

Then I tried to export, but I get the message "The File you chose could not be written to" If I select german, I get "Datei konnte nicht kopiert werden" which would be "Failed to copy file"

These are the translations for german:

```
<source>Failed to copy file</source>
    <translation>Datei konnte nicht kopiert werden</translation>
```

and 

```
    <source>The file you chose could not be written to.</source>
    <translation>Die gewählte Datei kann nicht überschrieben werden.
```

(By the way, the second should be "Die gewählte Datei kann nicht geschrieben werden")

so there seem to be a mixup in the line that is displayed in the error message, so I don't kno what went wrong.

I cannot save the profile now, thats the bug.

When I uncheck "make Tox protable" everything works fine again, so then there will only be left the mess up with the translation here :)

**Comment by dubslow** [comment]
You're sure of course that you're attempting to save to the same place both times?

And feel free to submit PR with any translation changes :)

**Comment by rubo77** [comment]
Yes, very sure, I tried lots of places on differend hdds and I have write-rights there.

the problem is gone if I deselect "make Tox protable" so it is a problem with the portability. you cannot export then. (I edited the title here accordingly)

**Comment by zetok** [comment]
@dubslow since you're still self-assigned to this.. are you by any chance working on this?

**Comment by photonometric** [comment]
Might this have been corrected along with (most) portability problems in commit 54c837a03e933fb164fb94647d36bb53fb58eb3a? 

Using a build from 4 days ago (b5b7bcfdd0f9734fd83a0c4a24b33100d53d07ae) I was able to create persistent portable qTox instances (in which "Make Tox Portable" is always ticked), and I seem to have no trouble exporting my profile. This .tox export also imports back into qTox successfully.
